### DISCUSSION TITLE
consile-filter

### DISCUSSION THREAD

**Discussion by helomen** [EssentialsX/Essentials#4746] [Q&A] [answered]
First of all, thank you for your great work! The plugin it really great! I just wanted to ask what the syntax is for the console-filter (EssentialsDiscord) when you want to filter out multiple regex patterns. For example multiple player names like "player1" "gamer2" "bot3". 
Thanks for your help.

**Comment by JRoy** [comment, marked as answer]
> First of all, thank you for your great work! The plugin it really great! I just wanted to ask what the syntax is for the console-filter (EssentialsDiscord) when you want to filter out multiple regex patterns. For example multiple player names like "player1" "gamer2" "bot3". Thanks for your help.

console-filter will also accept a list of regex patterns.

you could so something like:

```yml
console:
  ...
  console-filter:
    - "pattern 1"
    - "pattern 2"
```
